xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <!-- Match Mac OS Classic first due to weak matching on Mac OS X -->
+  <fingerprint pattern="^(?i:(?:Apple )?Mac OS ([7-9](?:\.\d+?)*?))$">
```

Comment:
The primary reason I came up with this is that I was testing our API usage of recog, and I expected this to be parsed. Perhaps its not something that would generally be "collected" of an asset, but certainly I can see it being input by a human (if don't manually). Perhaps I am crazy ;)